build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath 'com.android.tools.build:gradle:8.3.1'
```

Comment:
Note that many developers use IDEA instead of Android Studio.

Latest IDEA does not still support the Android Gradle plugin 8.3.X.
IDEA works fine with Android Gradle plugin 8.2.X (currently `8.2.2`).

Reply:
I guess we don't use any features from 8.3.x, so switching back to 8.2.2 shoudn't cause any issues.